### ISSUE TITLE
Cannot execute PowerShell scripts (No error message)

### ISSUE THREAD

**Issue by photon-fj** [PowerShell/vscode-powershell#4422]
### Prerequisites

- [X] I have written a descriptive issue title.
- [X] I have searched all [_open and closed_ issues](https://github.com/PowerShell/vscode-powershell/issues?q=is%3Aissue) to ensure it has not already been reported.
- [X] I have read the [troubleshooting](https://github.com/PowerShell/vscode-powershell/blob/main/docs/troubleshooting.md) guide.
- [X] I am sure this issue is with the _extension itself_ and does not reproduce in a standalone [PowerShell](https://github.com/PowerShell/PowerShell/issues/new/choose) instance.
- [X] I have verified that I am using the latest version of Visual Studio Code and the PowerShell extension.
- [X] If this is a security issue, I have read the [security issue reporting guidance](https://github.com/PowerShell/vscode-powershell/blob/main/SECURITY.md).

### Summary

Cannot execute PowerShell scripts with PowerShell Extension. No error is printed in the terminal, but no prompt appears. From the contents of EditorService.log it seems that there is a timeout in 5 minutes.

**TERMINAL**

> PowerShell Extension v2023.1.0
> Copyright (c) Microsoft Corporation.
> 
> https://aka.ms/vscode-powershell
> Type 'help' to get help.

**EditorServices.log**

> 2023-02-16 10:46:37.835 +09:00 [INF] ReadLine overridden with 'Microsoft.PowerShell.EditorServices.Services.PowerShell.Console.PsrlReadLine'
> 2023-02-16 10:46:37.846 +09:00 [VRB] Setting execution policy to "Restricted"
> 2023-02-16 10:46:37.848 +09:00 [VRB] Assembly resolve event fired for System.Management.Automation.resources, Version=3.0.0.0, Culture=ja-JP, PublicKeyToken=31bf3856ad364e35
> 2023-02-16 10:46:37.848 +09:00 [VRB] Assembly resolve event fired for System.Management.Automation.resources, Version=3.0.0.0, Culture=ja-JP, PublicKeyToken=31bf3856ad364e35
> 2023-02-16 10:46:37.848 +09:00 [VRB] Assembly resolve event fired for System.Management.Automation.resources, Version=3.0.0.0, Culture=ja-JP, PublicKeyToken=31bf3856ad364e35
> 2023-02-16 10:46:37.848 +09:00 [VRB] Assembly resolve event fired for System.Management.Automation.resources, Version=3.0.0.0, Culture=ja-JP, PublicKeyToken=31bf3856ad364e35
> 2023-02-16 10:46:37.875 +09:00 [VRB] Assembly resolve event fired for System.Management.Automation.resources, Version=3.0.0.0, Culture=ja-JP, PublicKeyToken=31bf3856ad364e35
> 2023-02-16 10:46:37.875 +09:00 [VRB] Assembly resolve event fired for System.Management.Automation.resources, Version=3.0.0.0, Culture=ja-JP, PublicKeyToken=31bf3856ad364e35
> 2023-02-16 10:46:37.877 +09:00 [VRB] Assembly resolve event fired for System.Management.Automation.resources, Version=3.0.0.0, Culture=ja-JP, PublicKeyToken=31bf3856ad364e35
> 2023-02-16 10:46:37.878 +09:00 [VRB] Assembly resolve event fired for System.Management.Automation.resources, Version=3.0.0.0, Culture=ja-JP, PublicKeyToken=31bf3856ad364e35
> 2023-02-16 10:51:37.525 +09:00 [DBG] Finished: Routing Request (0) initialize in 300011ms
> 2023-02-16 10:51:37.525 +09:00 [VRB] Request 0 was cancelled
> 2023-02-16 10:51:37.525 +09:00 [WRN] Tried to send request or notification before initialization was completed and will be sent later {"Id":0,"Error":{"Code":-32800,"Data":null,"Message":"Request Cancelled","$type":"ErrorMessage"},"Method":"initialize","$type":"RequestCancelled"}
> 2023-02-16 10:51:37.526 +09:00 [DBG] Finished: Processing request initialize 0 in 300018ms

**StartEditorServices.log**

> [DBG]: Creating/running editor services
> [DBG]: Creating startup info object
> [VRB]: Starting server, deregistering host logger and registering shutdown listener

**vscode-powershell.log**

> 2/16/2023 10:46:34 AM [NORMAL] - powershell.exe started.
> 2/16/2023 10:46:34 AM [NORMAL] - Waiting for session file...
> 2/16/2023 10:46:36 AM [NORMAL] - Session file found!
> 2/16/2023 10:46:36 AM [NORMAL] - powershell.exe PID: 10104
> 2/16/2023 10:46:36 AM [NORMAL] - Language server started.
> 2/16/2023 10:46:36 AM [NORMAL] - Connecting to language service on pipe: \\.\pipe\PSES_kh5z32pd.us4
> 2/16/2023 10:46:36 AM [NORMAL] - Session details: {"status":"started","languageServiceTransport":"NamedPipe","languageServicePipeName":"\\\\.\\pipe\\PSES_kh5z32pd.us4","debugServiceTransport":"NamedPipe","debugServicePipeName":"\\\\.\\pipe\\PSES_ml52ekbi.qik"}
> 2/16/2023 10:46:36 AM [NORMAL] - Language service socket connected.

### PowerShell Version

```console
Name                           Value
----                           -----
PSVersion                      5.1.22621.963
PSEdition                      Desktop
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0...}
BuildVersion                   10.0.22621.963
CLRVersion                     4.0.30319.42000
WSManStackVersion              3.0
PSRemotingProtocolVersion      2.3
SerializationVersion           1.1.0.1
```

### Visual Studio Code Version

```console
1.75.1
441438abd1ac652551dbe4d408dfcec8a499b8bf
x64
```

### Extension Version

```console
ms-vscode.powershell@2023.1.0
```

### Steps to Reproduce

Open VS Code, open any .ps1 file, error occurs every time

### Visuals

![vs](https://user-images.githubusercontent.com/92738383/219254041-52e1f603-5107-49d3-ab4b-653f0f125d8a.png)

**Comment by andschwa** [comment]
Does this repro with PowerShell 7.3?

**Comment by photon-fj** [comment]
Thank you for answering.

I installed PowerShell 7.3.2 and it worked fine.
The issue still occurs with PowerShell 5.1.

**Comment by andschwa** [comment]
Ok, that's helpful. Are you unblocked with PowerShell 7? Our PowerShell 5.1 support is best-effort, as it itself is in maintenance mode and it lacks most of the APIs we use do stuff (like execute given scripts) right. I'm wondering if you're seeing a duplicate of #3991 but it's really hard to say.

**Comment by SydneyhSmith** [comment]
Does the integrated terminal work at all? Does it come up with a prompt for you and return output?
